### PR TITLE
WIP, DO NOT MERGE: Feature/ape 404 check nominees

### PR DESCRIPTION
**Pull request by jsur**
WIP version of check-nominee.ts lambda function to check if `gql`, `zeus` etc. usage is what's wanted.

### CODE REVIEW

File: api/cron/check-nominee.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          sendSocialMessage({
+            message: `Nominee ${n.name} has only received ${n.nominations.length} vouch(es) and has failed`,
+            circleId: n.circle_id,
+          })
+        )
+      );
+      return res.status(200).json({ update_nominees });
+    }
+
+    res.status(200).json({ message: 'No updates' });
```
Comment (jsur) [inline api/cron/check-nominee.ts:86]: What do we want to return when there's no updated nominees?

File: src/lib/gql/zeusHasuraAdmin/index.ts [hunk 2]
```diff
@@ -16240,6 +16240,9 @@ const handleFetchResponse = (
 export const apiFetch =
   (options: fetchOptions) =>
   (query: string, variables: Record<string, any> = {}) => {
+    if (!!process.env.NODE_ENV) {
+      globalThis.fetch = nodeFetch as unknown as typeof fetch;
```
Comment (zashton) [inline src/lib/gql/zeusHasuraAdmin/index.ts:16244]: how about splitting this to a separate PR ?

File: api/cron/check-nominee.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+              _eq: false,
+            },
+            expiry_date: { _lte: new Date().toISOString() },
+          },
+        },
+        {
+          id: true,
+          name: true,
+          circle_id: true,
+          nominations: [{}, { id: true }], // TODO: aggregate query somehow?
```
[the remark this answers is not in the document]
Comment (zashton) [reply api/cron/check-nominee.ts:47]: you could use
`nominations_aggregate {
      aggregate {
        count
      }
    }
`

File: api/cron/check-nominee.ts (hunk 1 above)
Comment (zashton) [reply api/cron/check-nominee.ts:86]: its just saved inside logs, we could just keep it as no updates for now.

Comment (jsur) [reply api/cron/check-nominee.ts:86]: Ok!

File: api/cron/check-nominee.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (jsur) [reply api/cron/check-nominee.ts:47]: Great, that works, thanks.

File: src/lib/gql/zeusHasuraAdmin/index.ts (hunk 2 above)
Comment (jsur) [reply src/lib/gql/zeusHasuraAdmin/index.ts:16244]: Sure why not.

Comment (zashton) [reply src/lib/gql/zeusHasuraAdmin/index.ts:16244]: that will be great thanks! will def help unblock others

**Pull request closed without merging**